package.json

```diff
@@ -3431,7 +3431,7 @@
         }
     },
     "scripts": {
-        "vscode:prepublish": "npm run clean && npm run buildScripts && npm run lint && webpack --mode production && npm run copyFiles -- --webpacked",
```

Comment:
Removing lint from this command since linting errors will be caught in the new linting CI tests